build.sc

```diff
@@ -19,7 +19,7 @@ val scalaJSVersions = scalaVersions.map((_, "1.10.1"))
 val scalaNativeVersions = scalaVersions.map((_, "0.4.7"))
 
 trait MainArgsPublishModule extends PublishModule with CrossScalaModule with Mima {
-  def publishVersion = VcsVersion.vcsState().format()
+  def publishVersion = "0.5.0-M1"
```

Comment:
Why is that? Once you tag the commit, it should happen automatically.

Reply:
reverted